src/benchmarks/micro/libraries/System.Utf8String/Perf.Utf8String.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using System.IO;
+
+namespace System.Text
+{
+    [BenchmarkCategory(Categories.Libraries, Categories.Runtime)]
+    public class Perf_Utf8String
```

Comment:
@pgovind this will affect the changes I submitted into your branch. I'll adjust them after this is merged.

Reply:
Oh I just noticed those changes were already merged, since the base class is included (deleted) in this PR. Nevermind then.